src/main/kotlin/eu/europa/ec/eudi/pidissuer/adapter/out/pid/EncodePidInSdJwtVc.kt

```diff
@@ -213,3 +218,15 @@ private object Printer {
         return str
     }
 }
+
+/**
+ * Converts an [IsoGender] to an [OidcGender].
+ */
+private fun IsoGender.toOidGender(): OidcGender =
```

Comment:
Please add a new attribute to PID, that models gender as a string. Let the authoritative source to define its value, instead of mapping from ISO.